metricbeat/module/golang/fields.go

```diff
@@ -31,5 +31,5 @@ func init() {
 
 // Asset returns asset data
 func Asset() string {
-	return "eJzkV9tu40YMffdXEHnKokk+wEAX2Lqo+9CmBdIWBYrCoCVKmmYuwpCyrb8vZiQ5jiJp5WwCF1s9KSOHPOfwMuQtPFK9hNxptPkCQJRoWsLVOh5cLQBS4sSrUpSzS/i4AABoPoJxaaVpAcCF87JJnM1UvoQMNYdTT5qQaQlbEgy/IhFlc17CX1fM+urvBUCmSKe8jFZvwaKhEyzhkboMR95VZXsygOe5nVNbdCh36I/HQ/ZGbTbPCwt9Wt3Th3AKIzGpVpaefeuwPFK9dz7tfZtAFJ7fCuqMgstACsWtblB6l3s0wIJeYK+kuHshS0FYfokowX3PXTAJymbOGwz/EoRzTCls61bDu69ExEFweTKIq6/pDFRr9FvMCRKnNSXiPHBlDPr6rvfTIa1OIYkT1JsSK+5rNo1vBsZGvmAf1iuILiCtfBN4tyMPWmUkykRhS+8SYu7jn+JwysPysP+WgnbHZvEaFgDwfYdcWbgfgPlc0MRVVkYFHUEzX09bmS35INt6BXtkKLAsyVI6jsvSQTZ5stHKqLORNQW7hG0tNCT0HOT3dJAuX4OOe6V1ixv2BVn4kbD8pLVL4GNTZmiCiuOUkrLaZB6juVFGmXY4xHcO5B9a40Ho1a+/Q8zVqu1X69U4sverp5+Q5WU5cXhTNgcpCIyzSlz8sySv3EBSzKupsSSGNywrWAUnbSb3esSRk+IucSYYPeHmykyiHtP/HNhjfe1czNORgDkdDuaE4xxuM7rdKTCDh3eX+2c8fL1in6U27sa8vp3an3bkw3Tx/1N8cGzjmoX6LeXVo1u45rpp7WZfqKRortV4j7utoLKUQuadaR2/aqS7wA3ftMSGyxAPuObKhE7/5+HwUDNsSbv9h/FLtLNxASp/KIxxeqj5ZoLRXVxoNlwzfNu8qlQTfNO+24ppnB4LJo8X4PZdpNNNMREFYBi7UJy/AbRpmCKYzszKJ2Lt2naJuDXcOgAgLg5Evzx8psBb+mGQetMqP7H72v3MRBtDcnzpKnF/3CFaH+MRzTwNRuTtIEQPU63gH0rkPSD096k2ZJR2Pj+z6V0syZ+AXtOOLKhGxPQDSOFdlReukpj93ZI9TuRo6j9AJrQf6wRqkoYPXDMaAmRodkPcuh1N3BqhA1+Mh7LRP3CJdiJzwla5uyhK6+ztEend4t8AAAD//8IfSX8="
+	return "eJzkV91u60QQvs9TjHrVI9o+QCSOdAgiXEBBKiAkhKKJPbaX7u5YO+skfnu0aztNXdvH6WkVdPBVsrZmvu+bn525hUeql5CzRpsvALzympZwtY4HVwuAlCRxqvSK7RI+LgAAmpdgOK00LQCkYOc3CdtM5UvIUEs4daQJhZawJY/hK/Je2VyW8NeViL76ewGQKdKpLKPVW7Bo6ARLeHxdhiPHVdmeDOB5bufUFh3KHbrj8ZC9UZvN88JCn1b39CGcwkhMqpWlZ+86LI9U79mlvXcTiMLzW0GdUeAMfKEE1gyl49yhAfHoPOyVL+5eSFIQll8iSHB94iqYA2UzdgbD50EwFkphW7fa3X0F4g0Cy5NBTH0tZyBao9tiTpCw1pR4diCVMejqu96nQzqdQvLsUW9KrKSv1zS+GRgb6YJ9WK8guoC0ck3QeUcOtMrIKxNFLR0nJNLHP8XhlIeVYf8tBc3HBvEaFgDwfYdcWbgfgPlc0IQr60cFHUEzX09bmS25INt6BXsUKLAsyVI6jsvSwW/yZKOVUWcja4p1Cdva05DQc5Df08F3+Rp03CutW9ywL8jCj4TlJ605gY9NiaEJKo5TSspqkzmM5kYZZZpxiO8cyD+0xoPQq19/h5irVdur1qtxZO9XTz+h+JflJOGXsjn4gsCwVZ7j35Kc4oGkmFdTY0kMb1hWsApO2kzu9YgjJyVd4kwwesItlZlEPab/ObDH+tq5mKcjAXM6HMwJxzncZnS7U2AGD+8u9894+HrFPktt3I15fTu1P+3Iheni/6f44NgmtXjqt5RXj27hmuumtZt9oZKiuVbjPc5bj8pSCplj0zp+1Uh3gRu+aYkNlyEecC2VCZ3+z8PhoRbYkub9h/FLtLNxASp/KIxxeqjlZoLRXVxmNlILfNv8VKkm+Kb9bSuhcXriMXm8ALfvIp1uiokoAMPYhZ7dDaBNwxQhdGZWPhFrV7ZLxK3h1gEAz3Eg+uXhMwXe0g+D1JtW+Ynd1+5nJtoYkuNLV4n74w7R+hiPaOZoMCJvByF6mGoF/1Di3wNCf59qQ0Zp5/Mzm97FkvwJ6DXtyIJqREw/gC8cV3nBlY/Z3y3Z40SOpv4DZEL7seyhJt/wgWtBQ4ACzW6IW97RxK0ROvDFeCgb/YOUaCcyJ2yVu4uitGxvj0jvFv8GAAD//zhDRDk="
```

Comment:
@urso This file gets updated when I run `make update`. Not sure if it's OK to accept this change or why it happens.

Reply:
@ruflin @jsoriano Can you have a look? Is this file out of date? If so, we should update it in a separate PR.

Reply:
The reasons this gets updated is because the fields.yml gets updated below. It must be updated as part of this PR.